openapi/publicapi_spec.json

```diff
@@ -2660,6 +2667,11 @@
             "type" : "string",
             "description" : "The parameter in your request that's causing the error.",
             "example" : "amount"
+          },
+          "header" : {
+            "type" : "string",
+            "description" : "The header in your request that's causing the error.",
```

Comment:
Worth double checking this wording with Nathan if you haven't already

Reply:
👍